### PR TITLE
Update GDevelop interface page

### PR DESCRIPTION
**Pull request by LuniMoon**
Reorganising the structure of the article to:
- Separate the UI in 2 main sections
- List the parts on each section
- Send the user to specialised documentation

### CODE REVIEW

**Comment by LuniMoon** [comment]
Updated version for content (text and image) revision:
https://gdevelop-wiki-git-lunimoon-update-gdevelop-inte-780f1c-gdevelop.vercel.app/gdevelop5/interface/

Notes:

- The link to the "Create" section still needs to be settled and it's dependent on https://github.com/GDevelopApp/GDevelop-documentation/pull/246
- I'd like to get an image of the Education interface that displays the classrooms with a Sub.

File: docs/gdevelop5/interface/index.md [hunk 1]
```diff
@@ -1,65 +1,139 @@
...
+### 6. Play
+The Play section showcases community-made games published on gd.games, including multiplayer titles. [Sending written feedback](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/player-feedback/#giving-feedback) to creators through GDevelop's player feedback interface gives [credits](/gdevelop5/interface/profile/credits/) in exchange. These credits are redeemable in the Shop.
 
-When a project is opened, you can click on the top-left button to access to the project manager.
+![Home-play](Home-play.png)
 
-![](/gdevelop5/interface/pasted/20230309-173922.png)
 
-The project manager shows the structure of your game:
+## Project tab
```
Comment (4ian) [inline docs/gdevelop5/interface/index.md:70]: My issue is that this is not really something that exists in GDevelop. There is no "Project" tab anywhere?

Comment (LuniMoon) [reply docs/gdevelop5/interface/index.md:70]: True. A project is different tabs.
I've used "Project section" ? I don't know if "Project group" is better?
```suggestion
## Project section
```

File: images/Manage-Games-Dashboard.png
Comment (D8H) [inline images/Manage-Games-Dashboard.png:1]: This image is not used and duplicated from another folder. I guess it can be removed.

Comment (LuniMoon) [reply images/Manage-Games-Dashboard.png:1]: Should I delete it permanently?
I ignore if there is a way to know where else it is used (if that's the case). I confess these lists are not intuitive for me...
Screenshot 2024-12-30 at 15 58 26

Comment (D8H) [reply images/Manage-Games-Dashboard.png:1]: It was the only use of the image:

image

**Pull request merged**